### PR TITLE
[ADD] hr : Add configuration settings

### PR DESCRIPTION
**Pull request by ghost**
Task : https://www.odoo.com/web?#id=21703&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720

Pad : https://pad.odoo.com/p/r.aQ8dxoQt3CRF0RbD

### CODE REVIEW

File: addons/hr/res_config_view.xml [hunk 1]
```diff
@@ -6,5 +6,31 @@
...
+        </record>
+        <record id="action_hr_configuration_settings" model="ir.actions.act_window">
+            <field name="name">Configure Human Resources</field>
+            <field name="res_model">hr.config.settings</field>
+            <field name="view_mode">form</field>
+            <field name="target">inline</field>
+        </record>
+
+        <menuitem id="menu_hr_global_settings" name="Settings"
+            parent="menu_human_resources_configuration" sequence="0" action="action_hr_configuration_settings"/>
```
Comment (atp-odoo) [inline addons/hr/res_config_view.xml]: need to add group ==> groups="base.group_configuration"

File: addons/hr/res_config.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class HrConfigSettings(models.TransientModel):
+    _name = 'hr.config.settings'
+    _inherit = 'res.config.settings'
+
+    module_hr_gamification = fields.Boolean('Evaluate and motivate the employees', help='The Gamification module provides ways to evaluate and motivate the employees')
```
Comment (bst-odoo) [inline addons/hr/res_config.py]: Change help string into : This app gives you tools to challenge employees to reach specific targets. Goals are assigned through challenges to evaluate and compare team members with each other and through time.

Comment (rka-odoo) [inline addons/hr/res_config.py]: Done.

File: addons/hr/res_config.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+from odoo import fields, models
+
+
+class HrConfigSettings(models.TransientModel):
+    _name = 'hr.config.settings'
+    _inherit = 'res.config.settings'
+
+    module_hr_gamification = fields.Boolean('Evaluate and motivate the employees',
+    help='This app gives you tools to challenge employees to reach specific targets.\n'
```
Comment (tde-banana-odoo) [inline addons/hr/res_config.py]: Could you use a linter to avoid indentation issues ?

File: addons/hr/res_config.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    'Goals are assigned through challenges to evaluate and compare team members with each other and through time.')
+    module_hr_contract = fields.Boolean("Manage employees contracts",
+     help='Add all information on the employee form to manage employees contracts \n'
```
Comment (tde-banana-odoo) [inline addons/hr/res_config.py]: Could you use a linter to avoid indentation issues ?

File: addons/hr/res_config_view.xml [hunk 5]
```diff
@@ -6,5 +6,39 @@
             <field name="action_id" ref="base.action_open_website"/>
             <field name="state">open</field>
         </record>
+        <record id="view_hr_configuration" model="ir.ui.view">
```
Comment (tde-banana-odoo) [inline addons/hr/res_config_view.xml]: Please read the guidelines.

File: addons/hr/res_config_view.xml [hunk 6]
```diff
@@ -6,5 +6,39 @@
...
+                        <label for="id" string="Employee Contracts"/>
+                        <div>
+                            <field name="module_hr_contract"/>
+                            <label for="module_hr_contract"/>
+                        </div>
+                    </group>
+                </form>
+            </field>
+        </record>
+        <record id="action_hr_configuration_settings" model="ir.actions.act_window">
```
Comment (tde-banana-odoo) [inline addons/hr/res_config_view.xml]: Please read the guidelines.

File: addons/hr/res_config.py [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class HrConfigSettings(models.TransientModel):
```
Comment (tde-banana-odoo) [inline addons/hr/res_config.py]: Please see guidelines on how to name a file.

File: addons/hr/res_config_view.xml [hunk 8]
```diff
@@ -6,5 +6,42 @@
             <field name="action_id" ref="base.action_open_website"/>
             <field name="state">open</field>
         </record>
+        <record id="hr_config_settings_view_form" model="ir.ui.view">
```
Comment (atp-odoo) [inline addons/hr/res_config_view.xml]: change xml file name also. hr_config_settings_views.xml

File: addons/hr/res_config_view.xml [hunk 9]
```diff
@@ -6,5 +6,42 @@
...
+            <field name="name">Configure Human Resources</field>
```
Comment (atp-odoo) [inline addons/hr/res_config_view.xml]: change name as per new guidelines

**Pull request closed without merging**